jsonld/projects/C01-verified-carbon-standard-project.json

```
@@ -16,13 +16,14 @@
     "@type": "schema:URL",
     "@value": ""
   },
+  "regen:projectDescription": "",
```

why not just use the existing https://schema.org/description:
```suggestion
  "schema:description": "",
```